plugin.json

```diff
@@ -1 +1,31 @@
\ No newline at end of file
+{
+    "pluginmetadataversion": 2,
+    "name": "Snippet UI Plugin",
+    "type": [
+        "ui"
+    ],
+    "api": [
+        "python2",
+        "python3"
+    ],
+    "description": "Powerful code-editing plugin for writing and managing python code-snippets with syntax highlightingd, hotkey binding and other features",
+    "longdescription": "The snippet editor started as a simple example UI plugin to demonstrate new features available to UI plugins. It has turned into a functionally useful plugin in its own right. The snippet editor allows you to write small bits of code that might not be big enough to warrant the effort of a full plugin but are longer enough that you don't want to retype them every time in the python-console!\n\nAs an added bonus, all snippets are added to the snippets menu and hot-keys can be associated with them as they make use of the action system. All action-system items are also available through the command-palette (CTL/CMD-p).\n\n![](https://github.com/Vector35/snippets/blob/master/media/snippets.gif?raw=true)\n\n.",

```

Comment:
"longer enough" should be "long enough"

Reply:
Yeah, but that's not a new bug == my original text had that wrong. This is just more of a formatting issue. Which actually reminds me that I need to update https://github.com/Vector35/release_helper/blob/master/do_release.py#L40 to have an `indent=` parameter which will prevent this from regressing the next time I use it.